templates/cdk-erigon/chainspec.json

```diff
@@ -0,0 +1,24 @@
+{
+  "ChainName": "dynamic-kurtosis",
```

Comment:
nit: consider more descriptive name i.e. cdk-erigon-devnet

Reply:
The `dynamic` prefix is mandatory for cdk-erigon to work at the moment unfortunately.
For reference: https://github.com/0xPolygonHermez/cdk-erigon?tab=readme-ov-file#dynamic-chain-configuration